### PR TITLE
Project News site magazine 

### PR DESCRIPTION
**Pull request by JHarde**
https://footballnewsproject.netlify.app/

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -2,21 +2,73 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Football News</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="icon" type="image/png" href="./">
```
Comment (bealun) [inline code/index.html:7]: Was this about to become a favicon? Next project? 😎

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,73 @@
...
+    <!--Header with logo and navbar-->
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="logo-name">
+        <img class="logo" src="./images/football.jpg" alt="logo">
+        <h1 class="website-name">Football News</h1>
+        </div>
+        <nav>
+          <ul>
+          <li class="nav-logo"><a href="https://www.fotbollskanalen.se/" target="_blank"><img src="./images/fk_logo.png" alt="Fotbollskanalen logo"/></a></li>
```
Comment (bealun) [inline code/index.html:21]: Really nice to open links in new tab 👏

File: code/index.html [hunk 3]
```diff
@@ -2,21 +2,73 @@
...
+          <li class="nav-logo"><a href="https://www.bbc.com/sport/football" target="_blank"><img src="./images/bbc-sport-logo.png" alt="BBC logo"/></a></li>
+          </ul>
+      </nav>
     </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+    <!--Big news section-->
+    <section class="big-news-container">
+      <div class="news-card-image"><img src="./images/messi.jpg" alt="Lionel Messi"/></div>
```
Comment (bealun) [inline code/index.html:30]: Nice structure with image folder!

File: code/index.html [hunk 4]
```diff
@@ -2,21 +2,73 @@
...
+    <!--Other news section-->
+    <section class="other-news-container">
+      <div class="news-child"><img src="./images/koeman.jpg" alt="Ronald Koeman"/><p>New Barcelona coach Ronald Koeman</p></div>
+      <div class="news-child"><img src="./images/cristiano.jpg" alt="Cristiano Ronaldo"/><p>Messi's biggest rival Cristiano Ronaldo</p></div>
+      <div class="news-child"><img src="./images/family.jpg" alt="The Messi family"/><p>Is the Messi family looking for a new home?</p></div>
+      <div class="news-child"><img src="./images/messi-city.jpg" alt="Messi in Man City?"/><p>Could we be seeing Messi in Man City?</p></div>
+      <div class="news-child"><img src="./images/pepaguero.jpg" alt="Reason to move?"/><p>This could be a reason to move to City</p></div>
+      <div class="news-child"><img src="./images/messiaguero.jpg" alt="Good friends"/><p>Messi and Agüero are close friends</p></div>
+      <div class="news-child"><img src="./images/messi-dad.jpg" alt="Messi's father"/><p>Messi's father and agent Jorge Messi</p></div>
+      <div class="news-child"><img src="./images/messiballon.jpg" alt="Ballon dór"/><p>The one and only six time winner of Ballon dór</p></div>
```
Comment (bealun) [inline code/index.html:59]: These could use some indentation and structure however 🙃 But good use of alt in the image tags!!

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,247 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*  {
+  box-sizing: border-box;
```
Comment (bealun) [inline code/style.css:2]: BRAVO!! So you don't need to repeat code 💫

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,247 @@
...
+
+
+}
+ p  {
+  font-size: 20px;
+  font-family: 'Josefin Sans';
+  padding: 20px;
+}
+
+.news-child >p:hover  {
```
Comment (bealun) [inline code/style.css:24]: Gotta love hover effects 🥳

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,247 @@
...
+  align-items: center;
+  justify-content: center;
+  margin: 15px;
+  padding: 15px;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.news-card-text  {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```
Comment (bealun) [inline code/style.css:126]: Really nice UI feature 🙌

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,247 @@
...
+.news-child  {
+  margin:10px;
+  box-shadow: 5px 5px 5px 0px rgba(0,0,0,0.75);
+  border: 0.5px solid;
+  width: 350px;
+  height: auto;
+  max-width: 350px;
+}
+
+.news-child:hover  {
```
Comment (bealun) [inline code/style.css:167]: This is so good 👏👏👏

**Review by bealun — CHANGES_REQUESTED** [review]
This site will turn anyone into a football fan! You've done a really nice job making everything look good with the hover effects, the little animation and the header links 👏 
There are some smaller changes requested regarding the tablet/iPad styling for your site. The header goes outside your content but it's an easy fix if you inspect and maybe wrap whatever it is that's making it push out. Have a look and then you'll be approved!
Still amazing work this week and be really proud of yourself 🎉

**Review by JHarde — COMMENTED** [review]
Have made some minor changes now, let me now if there is anything else I need to change.

**Review by bealun — APPROVED** [review]

**Pull request closed without merging**